frontend/src/components/RestaurantDetails/RestaurantInfo.js

```diff
@@ -42,23 +46,23 @@ class restaurantInfo extends React.Component {
                         image="https://www.omnihotels.com/-/media/images/hotels/bospar/restaurants/bospar-omni-parker-house-parkers-restaurant-1170.jpg"
                         title={"ok"}
                     />
-                    <Typography gutterBottom variant="h5" component="h2">
+                    <Typography gutterBottom variant="h3">
                         {restInfo.name}
                     </Typography>
-                    <Typography component="p">
+                    <Typography component="p" variant="h5">
                         Address: {restInfo.addres_id} <br/>
                         Works hour: 10.00 - 23.00 <br/>
                         Phone: 093-999-23-08
-                        <Button component={Link} to={'menu/' + restInfo.id} variant="outlined" className={classes.button}>
-                            Watch menu
-                        </Button>
                     </Typography>
                     <Divider variant="middle" style={{marginTop:"10px"}}/>
-                    <Typography component="p">
+                    <Typography component="p" variant="h5">
                         {restInfo.description}{restInfo.description}
                     </Typography>
-
-
+                    <Typography align="right" style={{margin:10}}>
+                        <Button component={Link} to={'menu/' + restInfo.id} variant="contained" color="primary">
+                            Watch menu
```

Comment:
"See menu" is better I guess. For reference: https://www.gloriafood.com/order-online-button-customization